config/config.go

```diff
@@ -72,6 +74,7 @@ func ParseConfigOptions(cfgOptions ...Option) *Config {
 		Daemon: DaemonCfg{
 			DriverCfg:       make(map[string]interface{}),
 			DisableProvider: make(chan struct{}, 10),
+			ExecRoot:        "/var/run/docker/",
```

Comment:
We should not have any linux path defaults in `config.go` since this is supposed to be OS agnostic. In fact we don't need to have any defaults here are at all.